services/minespace-web/src/components/dashboard/mine/projects/NoticeOfWorkProjects.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC, useContext, useEffect, useState } from "react";
+import { useSelector } from "react-redux";
+import { Row, Col, Typography } from "antd";
+import { SidebarContext } from "@mds/common/components/common/SidebarWrapper";
+import { IMine } from "@mds/common/interfaces/mine.interface";
+import NoticeOfWorkTable from "./NoticeOfWorkTable";
+import { getNoticeOfWorkList } from "@mds/common/redux/selectors/noticeOfWorkSelectors";
+import { fetchProponentNoticeOfWorkApplications } from "@mds/common/redux/actionCreators/noticeOfWorkActionCreator";
+import { useAppDispatch } from "@mds/common/redux/rootState";
+
+export const Projects: FC = () => {
```

Comment:
Is this component intended to show projects other than NoticeOfWorkProjects down the line? If not, something more descriptive will probably be helpful.